src/middlewares/types.ts

```diff
@@ -1,10 +1,17 @@
-import { validateSign } from './authValidation';
-import { validatePostBody } from './postValidations';
-import { validateUserBody, validateUserId } from './userValidation';
+import { validateSign } from "./authValidation";
+import { validatePostBody, validateUpdatePostBody } from "./postValidations";
+import {
+  validateUpdateUserBody,
+  validateUserBody,
+  validateUserId,
+} from "./userValidation";
 
 export type validateUserBody = typeof validateUserBody;
```

Comment:
Please rename somehow `type validateUserBody` and all others. A type name should not be the same as the function name